backend/imap/src/test/java/com/fsck/k9/backend/imap/ImapSyncTest.kt

```diff
@@ -151,7 +157,8 @@ class ImapSyncTest {
 
         imapSync.sync(FOLDER_SERVER_ID, defaultSyncConfig, syncListener)
 
-        assertThat(backendFolder.getMessageServerIds()).containsExactly("9", "13")
+        // FIXME Does the order matter? If yes, we need to fix ImapFolder: containsExactly -> containsOnly
```

Comment:
Truth's `containsExactly` doesn't check the item order. So it's generally a mistake to replace it with assertk's `containsExactly`. Most of the time it should probably be replaced by `containsExactlyInAnyOrder`.

Also, if a method returns a `Set` the order generally doesn't matter. If it ever does, we'll probably return a special subclass of `Set`.